ams/server/src/main/java/com/netease/arctic/server/manager/FlinkOptimizerContainer.java

```diff
@@ -416,6 +420,8 @@ public static class FlinkConfKeys {
     public static final String JOB_MANAGER_TOTAL_PROCESS_MEMORY = "jobmanager.memory.process.size";
     public static final String TASK_MANAGER_TOTAL_PROCESS_MEMORY =
         "taskmanager.memory.process.size";
+    public static final String USER_LIB_FIRST = "yarn.per-job-cluster.include-user-jar";
```

Comment:
`USER_LIB_FIRST` may not be an approprite name. How about using the same property name [CLASSPATH_INCLUDE_USER_JAR](https://github.com/apache/flink/blob/d5b1afb7f67fe7da166193deb7711b8a8b163bcf/flink-yarn/src/main/java/org/apache/flink/yarn/configuration/YarnConfigOptions.java#L57C1-L57C1) from Flink?